.github/workflows/ci.yml

```diff
@@ -11,14 +11,20 @@ jobs:
     strategy:
       matrix:
         system: [macos-latest, pqcp-arm64]
+        skipped: [false]
+        include:
+        - system: ubuntu-latest
```

Comment:
Just to be sure I understand the meaning of this: It means we add `ubuntu-latest` to the `system` array, and set `skipped` to `false` for `macos-latest` and `pqcp-arm64`, but to `true` to `ubuntu-latest`?

Reply:
yes, exactly